src/K8sClusterManagers.jl

```diff
@@ -1,19 +1,13 @@
 module K8sClusterManagers
 
-using Distributed
+using Distributed: Distributed, ClusterManager, WorkerConfig, cluster_cookie
```

Comment:
```suggestion
using Distributed
```
Those are already exported from Distributed

Reply:
I know. When it's not too much of a pain I like my libraries to be very explicit

Reply:
Being explicit is much nicer for those unfamiliar with the exports of the dependency. Makes it much more obvious where functionality is coming from

Reply:
Eh, don't really agree but  ¯\\\_(ツ)\_/¯

Reply:
Another good reason to do this is that it makes it very obvious when dependencies are no longer used.

Reply:
I definitely wouldn't advocate doing this everywhere